packages/modular-scripts/src/esbuild-scripts/utils/createViewTrampoline.ts

```diff
@@ -0,0 +1,66 @@
+import * as esbuild from 'esbuild';
+import { createRewriteDependenciesPlugin } from '../plugins/rewriteDependenciesPlugin';
+import type { Dependency } from '@schemastore/package';
+
+export const indexFile = `
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="static/js/_trampoline.js"></script>
+  </body>
+</html>
+`;
+
+export async function createViewTrampoline(
+  fileName: string,
+  srcPath: string,
+  dependencies: Dependency,
+  browserTarget: string[],
+): Promise<esbuild.BuildResult & { outputFiles: esbuild.OutputFile[] }> {
+  const fileRelativePath = `./${fileName}`;
```

Comment:
Surely this would be the package name?

Reply:
This is `index.js` if we're starting a view or `index-[HASH].js` if we're building. I think that the actual view entrypoint file should be hashed for cache-breaking purposes? After all whoever loads it fetches it at runtime, not downloads it from a registry and process it at build time, like the previous views. We keep track of the name in the output `package.json`, `module` field, so whoever loads it has a way to get the file name. If you think it should be named differently let's have a chat about that.